### DISCUSSION TITLE
Bagaimana memunculkan bulan dalam abbr pada format m/y

### DISCUSSION THREAD

**Discussion by AjiPramasto** [teamalgoritma/community#240] [Q&A] [answered]
Halo,

Mohon bantuannya code untuk memunculkan bulan dalam abbr pada format m/y pada kolom Month sbb:
![image](https://user-images.githubusercontent.com/107389614/182398011-e5bf634f-b5d7-4c2a-b17b-22c35e59cbb6.png)

terima kasih

**Comment by inytss** [comment, marked as answer]
Dear @AjiPramasto ,
Berikut cara yang bisa Bapak coba untuk mengubah format bulan/tahun dalam bentuk nama-bulan/tahun.
Bapak dapat memanfaatkan fungsi `as.yearmon()` dari package `zoo` dimana disitu Bapak bisa memasukkan format penulisan data yang Bapak miliki. 
Sebagai contoh:
```
df <- c("01/19", "08/19", "04/17")
zoo::as.yearmon(df, "%m/%y")
```
maka output dari data tersebut adalah sebagai berikut
```
[1] "Jan 2019" "Aug 2019" "Apr 2017"
```

Demikian jawaban yang dapat saya berikan, semoga membantu. Apabila dirasa jawaban ini membantu, silahkan tekan tombol "_Mark as aanswer_". Terima kasih.

**Reply by AjiPramasto** [reply]
Terima kasih bantuanya, masih belum bisa.
hasilnya NA, atau saya yang salah nulis kodenya yah:
![image](https://user-images.githubusercontent.com/107389614/182555935-d9ccc42f-785b-4d56-ba71-0e61da3622a0.png)

**Comment by AjiPramasto** [comment]
ok sudah bisa, raw datanya diperbaiki dulu,
terima kasih bantuannya
![image](https://user-images.githubusercontent.com/107389614/182562767-11d0a7fc-9953-412d-9e8f-627f36cc8703.png)

**Comment by kevbow** [comment]
Halo @AjiPramasto,

Anda juga bisa menggunakan fungsi `month()` dari package `lubridate` dan menggunakan beberapa parameter di dalamnya terkait nama bulan serta penggunaan singkatan. Tetapi sebelum digunakan, bentuk awalnya harus diubah terlebih dahulu dengan fungsi `my()` juga dari package `lubridate`. Berikut ini adalah contoh kode yang dapat saya lampirkan:
```
library(lubridate)
tgl <- c("01/17", "01/18", "02/17")

month(my(tgl), abbr = T, label = T)
```

Demikian yang dapat saya sampaikan, semoga membantu.
